### PR TITLE
Install MegaLinter

### PR DESCRIPTION
**Pull request by josecelano**
Testing a new linter. We are using super-linter rigth now.

### CODE REVIEW

**Comment by josecelano** [comment]
After testing it a little bit I think it's a better option. I like

- The report on the PR
- Easy installation and use
- You can run it locally without using act
- Totally decouple from GitHub Actions
- It helps you to setup IDE configuration
- Community driven

How to install and use:

```
npm install mega-linter-runner -g
mega-linter-runner --fix
```

File: .python-lint [hunk 1]
```diff
@@ -0,0 +1,544 @@
...
+
+# Number of spaces of indent required inside a hanging  or continued line.
+indent-after-paren=4
+
+# String used as indentation unit. This is usually "    " (4 spaces) or "\t" (1
+# tab).
+indent-string='    '
+
+# Maximum number of characters on a single line.
+max-line-length=100
```
Comment (da2ce7) [inline .python-lint:413]: set to 120 as configured in flake8

File: .gitignore [hunk 2]
```diff
@@ -1,3 +1,4 @@
 __pycache__/
 .coverage
-dist/
\ No newline at end of file
+dist/
+report/
```
Comment (da2ce7) [inline .gitignore:4]: add newline to the end of the file.

File: .isort.cfg [hunk 3]
```diff
@@ -0,0 +1,8 @@
+[settings]
+profile=
```
Comment (da2ce7) [inline .isort.cfg:2]: need to add the black profile here since we are using black for formatting.

**Review by da2ce7 — COMMENTED** [review]
Overall Looks Good :)

**Pull request merged**